governance/feishu-sync.py

```
@@ -0,0 +1,378 @@
+    def _tables_path(self):
+        return f"/open-apis/bitable/v1/apps/{self.app_token}/tables"
+
+    def _records_path(self, table_id):
+        return f"{self._tables_path()}/{table_id}/records"
+
+    def find_or_create_table(self, name, state_names):
+        """按表名找数据表；缺则创建（IFACE-04 字段+State 单选全集）——幂等，
+        整表删除后单轮同步内重建的机制基础（BEH-06）。"""
+        cur = None
+        while True:
+            q = f"?page_size=100" + (f"&page_token={cur}" if cur else "")
```

Ruff (0.16.2)

[error] f-string without any placeholders

Remove extraneous `f` prefix

(F541)